include/EASTL/sort.h

```diff
@@ -797,10 +818,10 @@ namespace eastl
 			if(compare(*i, *first))
 			{
 				EASTL_VALIDATE_COMPARE(!compare(*first, *i)); // Validate that the compare function is sane.
-				const value_type temp(*i);
-				*i = *first;
+				value_type temp(eastl::forward<value_type>(*i));
```

Comment:
Uh... don't remember why at this point... and I don't see why I did... added it back and everything compiles and benchmarks seem ok. Will add back.